tests/di/ContainerTest.php

```
@@ -173,4 +175,16 @@
         $this->expectException(Unbound::class);
         (new Container())->getInstanceWithArgs(FakeEngineInterface::class, []);
     }
+
+    public function testWeaveAspectsWithEmptyPointcuts(): void
+    {
+        $container = new Container();
+        (new Bind($container, FakeEngine::class));
+
+        // Should work fine even when no pointcuts are defined
+        $container->weaveAspects(new Compiler(sys_get_temp_dir()));
```

PHPStan (2.1.31)

Parameter #1 $classDir of class Ray\Aop\Compiler constructor expects non-empty-string, string given.


(argument.type)